launch/tier4_planning_launch/launch/scenario_planning/lane_driving/behavior_planning/behavior_planning.launch.py

```diff
@@ -246,12 +222,15 @@ def launch_setup(context, *args, **kwargs):
     launch_run_out_with_points_method = PythonExpression(
         [
             LaunchConfiguration(
-                "launch_run_out", default=behavior_velocity_planner_param["launch_run_out"]
+                "launch_run_out",
+                default="behavior_velocity_planner::RunOutModulePlugin"
+                in behavior_velocity_planner_params["launch_modules"],
+            ),
```

Comment:
From what I understand, the `launch_run_out` parameter is removed so I think this condition can be simplified by removing the `LaunchConfiguration`.

Reply:
I keep this because they are different variables with the same name. One is the parameter in behavior_velocity_planner node and another is the argument in launch file. But I also think that it is not used. Can I remove it? @TakaHoribe

Reply:
@maxime-clem I asked the maintainer of the module (@TomohitoAndo). The `launch_run_out` is not needed, so I remove it. https://github.com/autowarefoundation/autoware.universe/pull/3811/commits/53e8694327e77895b697fc5979596aae85cd09e9